### PR TITLE
Th/mongocontrollers

### PR DESCRIPTION
**Pull request by timdoozy**
Issue #5 and issue #11 have been addressed and shown to be working properly. Added card and order model schemas. Added sample cards to our database. Added mocha tests for existing CRUD functions. Added controllers for orders and cards. Lastly, added userapi routes.

### CODE REVIEW

File: server/tests/mongo_test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+//test if we can establish a connection to our mongodb
+const mongoose = require('mongoose');
+//tell mongoose to use es6 implementation of promises
+mongoose.Promise = global.Promise;
+
+mongoose.connect('mongodb://xxx/'); //need to add our uri when our db is made
```
Comment (synchronizing) [inline server/tests/mongo_test.js]: Why are we not using the projects' environment variables for the MongoDB here?

**Comment by timdoozy** [comment]
server/tests/mongo_test.js can be deleted. I forgot to delete that file because I made different tests within the server/models folder.

**Comment by timdoozy** [comment]
okay I got rid of that file.

**Comment by synchronizing** [comment]
Nope, looks good to me 👍🏼

**Pull request merged**